Makefile

```diff
@@ -22,11 +26,9 @@ rm-mocked:
 	grep -rl "^// Code generated by mockery" | grep .go$ | xargs -r rm
 
 .PHONY: generate
-generate: mockery install-protoc
-	# add our installed protoc to the head of the PATH
-	# maybe there is a cleaner way to do this
-	PATH=$$HOME/.local/bin:$$PATH go generate -x ./...; \
```

Comment:
@krehermann you touched this last - are you the original author? I think it seems reasonable to drop this and expect `GOBIN` to already be on the path, no?